<ui version="4.0" >
 <class>NewPlayer</class>
 <widget class="QDialog" name="NewPlayer" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Nuovo Giocatore</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>430</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>30</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cognome</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>70</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Nome</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit" >
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>30</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2" >
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>70</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>120</y>
     <width>141</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title" >
    <string>Ruolo</string>
   </property>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>97</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QRadioButton" name="radioButton" >
       <property name="text" >
        <string>Portiere</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_2" >
       <property name="text" >
        <string>Difensore</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_3" >
       <property name="text" >
        <string>Centrocampista</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_4" >
       <property name="text" >
        <string>Attaccante</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>350</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>310</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Squadra</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>310</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="inputMask" >
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>350</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Prezzo</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewPlayer</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewPlayer</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
